### DISCUSSION TITLE
Docs quickstart: break up Engineering Guidelines page

### DISCUSSION THREAD

**Discussion by dredlong** [twilio-labs/paste#1205] [Ideas] [answered]
Hi there - as a first-time Paste user I wanted to give some quick feedback on the quickstart guide.

For me it was a bit confusing that the quickstart ran straight into the manual install guide and a longer description of the engineering principles.

It would have been clearer for me if quickstart was a single page and, after the create React app step, it also documents running the initial template app locally and what you should expect to see if it has built correctly.

**Comment by gloriliale** [comment, marked as answer]
Hi @dredlong! We appreciate you taking the time to give us feedback, especially on the "Getting Started" step. To confirm, it would be more helpful if there was guidance after the create React step to validate that you installed it correctly and can safely move forward? I'll ask the team about having the quickstart be separate as well. Thanks!

**Reply by dredlong** [reply]
Yep, exactly. The first time I did it I assumed the whole page was quick start instructions so I ran straight into the manual install process from the template-based setup and then got confused.

It would help to have the point at which you run `yarn start` or `npm start` for the first time made clear and a screenshot of the template app would be great to compare with what the user sees after starting the app for the first time.

**Reply by gloriliale** [reply]
Excellent, thanks for expanding on that! This is definitely something we should do. I've made a ticket for breaking up the getting started page for legibility and adding more validation guidelines, and added it to our backlog: https://issues.corp.twilio.com/browse/DSYS-2636
